python/pacemaker/_cts/audits.py

```diff
@@ -287,13 +304,18 @@ def __call__(self):
         self._cm.ns.wait_for_all_nodes(self._cm.env["nodes"])
 
         for node in self._cm.env["nodes"]:
-            found = self._find_core_on_fs(node, "/var/lib/pacemaker/cores/*", "Pacemaker")
-            if found:
-                result = False
+            if self._cm.env["have_systemd"]:
+                found = self._find_core_with_coredumpctl(node)
```

Comment:
> If we're running on a system that supports systemd, core dumps will be
> managed using coredumpctl.

Systemd added `coredumpctl` in v195:
https://github.com/systemd/systemd/commit/5de0409

It also appears to be possible for systemd coredump handling to be disabled despite having systemd:
https://systemd.io/COREDUMP/

What you have here is probably good enough for CTS lab, and it's probably not worth the effort to be more thorough. Just want you to be aware.

> we would have to search for every pacemaker or
> corosync binary possible.

FWIW there aren't that many, right? Granted, we'd have to keep the list up-to-date if they change...

Reply:
> FWIW there aren't that many, right? Granted, we'd have to keep the list up-to-date if they change...

Well, it would be all the daemons and command line tools.  It seems like kind of a lot to me, and I don't really want to maintain a separate list somewhere.